desc/compute/_metric.py

```diff
@@ -1823,3 +1823,74 @@ def _gradtheta(params, transforms, profiles, data, **kwargs):
 def _gradzeta(params, transforms, profiles, data, **kwargs):
     data["|grad(zeta)|"] = jnp.sqrt(data["g^zz"])
     return data
+
+
+@register_compute_fun(
+    name="gbdrift",
+    label="\\mathrm{gradB-drift}",
+    units="~",
+    units_long="None",
+    description="Binormal component of th geometric part of the gradB drift"
+    + " used for local stability analyses,"
+    + " energetic particle proxy,"
+    + " effective field ripple",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
```

Comment:
doesn't look like grid is required for this

Reply:
Removed.